client/src/components/common/Questionnaire/MultiChoicesQuestion.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Checkbox } from 'antd';
+
+import './style.css';
+
+const MultiQuestion = props => {
+  const { onMultiChange, options, id, defaultValue, keyValue } = props;
+  return (
+    <div>
+      {options.question && <h1 className="qusetion">{options.question}</h1>}
```

Comment:
If you used destructuring above
```suggestion
      {question && <h1 className="qusetion">{question}</h1>}
```

Reply:
yeah I guess their is no reason to use logic operation